src/b_setup.hpp

```diff
@@ -238,6 +238,10 @@ void setup() {
   #endif
 // end microstepping -------------------
 
+  #if USE_HALL_SENSOR_RA_AUTOHOME == 1
+  pinMode(RA_HOMING_SENSOR_PIN, INPUT);
```

Comment:
is here indentation missing? at least it does not seem to match indentation of the serial line below.